### ISSUE TITLE
Building from source causes the URL:MAGNET protocol on Windows to be replaced and unchangable

### ISSUE THREAD

**Issue by minhngoc25a** [G-Ray/pikatorrent#137]
**Describe the bug**
This bug only affects power users who want to build PikaTorrent from source. It doesn't affect normal users using the stable build. After running the build commands for PikaTorrent, the URL:MAGNET protocol on Windows will be set to whenever PikaTorrent's executable is after being built.

**To Reproduce**
Steps to reproduce the behaviour: (assuming beforehand we had qBittorrent as the default application to process magnet links)
Step 1: Clone PikaTorrent's source code
Step 2: Run the build commands
```
npm run web # To Start the web server
npm run desktop # To start the electron app
```
Step 3: The URL:MAGNET protocol is modified, and now by default, PikaTorrent's development executable will handle it instead. Since PikaTorrent also has a Microsoft Store build, this might be the reason why, after the protocol is modified, we can only select Microsoft Store apps to handle magnet links (see picture below).

**Expected behaviour**
The default app selector should be able to select any type of app, not just limited to Microsoft Store apps but external applications as well.

**Screenshots**
![image](https://github.com/G-Ray/pikatorrent/assets/33834488/760c40b2-8481-44b4-8699-bc1e18146658)

**Platform:**
Windows

**Comment by G-Ray** [comment]
So if I understand correctly, `PikaTorrent` should set itself as the default magnet link handler only in production mode, and not in development, right ? I agree with this :100:

**Comment by minhngoc25a** [comment]
Yes. The only way I was able to remove that was to install the production PikaTorrent, then uninstall.

**Comment by G-Ray** [comment]
@minhngoc25a Have you retried the behavior with latest releases ?

**Comment by G-Ray** [comment]
Closing this as the app as been rewritten, behavior might be differente
